source/src_io/energy_dos.cpp

```diff
@@ -580,7 +553,7 @@ void energy::perform_dos(Local_Orbital_wfc &lowf, LCAO_Hamilt &uhm)
 					 const int m1 = atom1->iw2m[j];
 					 out <<std::setw(5) << i << std::setw(8) 
 						<< GlobalC::ucell.atoms[t].label <<std::setw(5)
-							<<L1<<std::setw(5) <<m1<<std::setw(5)<<N1+1<<std::setw(15)<< Name_Angular[L1][m1] << std::endl;
+							<<L1<<std::setw(5) <<m1<<std::setw(5)<<N1+1<<std::setw(15)<< ModuleBase::Name_Angular[L1][m1] << std::endl;
```

Comment:
Why do you move this label to module_base/constants.h?
If these labels only are only used in this file and you want to only define them once, you can move the code from constants.h to head of this file.
Will any other code use this labels in the future?

Reply:
I moved this label from constant.h to src_pw/energy.h in the new commit